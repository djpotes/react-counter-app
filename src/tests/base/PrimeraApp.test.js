import PrimerApp from '../../PrimeraApp';
//import {render} from '@testing-library/react';
import {shallow} from 'enzyme';

describe('Pruebas con PrimeraApp component', () => {

    /*test('Debe de mostrar el mensaje "Hola, Soy Goku" ', () => {
        const saludo = "Hola, Soy Goku";
        const {getByText} = render(<PrimerApp saludo = {saludo}/>);
        expect(getByText(saludo)).toBeInTheDocument(); 
    });*/

    test('Debe de mostrar <PrimerApp/> correctamente" ', () => {
        const saludo = "Hola, Soy Goku";
        const wrapper = shallow(<PrimerApp saludo={saludo}/>);
        expect(wrapper).toMatchSnapshot();
    });

    test('Debe de mostrar el subtitulo enviado por props', () => {
        const saludo = "Hola, Soy Goku";
        const subtitulo = "Soy un subtitulo";
        const wrapper = shallow(<PrimerApp saludo={saludo} subtitulo={subtitulo} />);
        const textoParrafo = wrapper.find('p').text();
        expect(textoParrafo).toBe(subtitulo);
    });
});