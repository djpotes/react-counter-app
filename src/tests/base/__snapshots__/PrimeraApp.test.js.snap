// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con PrimeraApp component Debe de mostrar <PrimerApp/> correctamente"  1`] = `
<Fragment>
  <div>
    Hola, Soy Goku
  </div>
  <p>
    Mi primera aplicaci√≥n
  </p>
</Fragment>
`;
